test/libdnf/base/test_goal.cpp

```diff
@@ -50,3 +50,46 @@ void BaseGoalTest::test_install() {
     CPPUNIT_ASSERT_EQUAL(0lu, obsoleted_set.size());
 }
 
+void BaseGoalTest::test_install_from_cmdline() {
+    libdnf::Goal goal(base.get());
+    std::filesystem::path repo_path = PROJECT_SOURCE_DIR "/test/libdnf/rpm/repos-data/dnf-ci-fedora/x86_64/wget-1.19.5-5.fc29.x86_64.rpm";
+    auto cmd_pkg = sack->add_cmdline_package(repo_path, false);
+    goal.add_rpm_install(cmd_pkg, true);
+    goal.resolve();
+    auto install_set = goal.list_rpm_installs();
+    auto reinstall_set = goal.list_rpm_reinstalls();
+    auto upgrade_set = goal.list_rpm_upgrades();
+    auto downgrade_set = goal.list_rpm_downgrades();
+    auto remove_set = goal.list_rpm_removes();
+    auto obsoleted_set = goal.list_rpm_obsoleted();
+    CPPUNIT_ASSERT_EQUAL(1lu, install_set.size());
+    CPPUNIT_ASSERT_EQUAL(install_set[0].get_full_nevra(), std::string("wget-0:1.19.5-5.fc29.x86_64"));
+    CPPUNIT_ASSERT_EQUAL(install_set[0].get_repo()->get_id(), std::string("@commandline"));
+    CPPUNIT_ASSERT_EQUAL(0lu, reinstall_set.size());
+    CPPUNIT_ASSERT_EQUAL(0lu, upgrade_set.size());
+    CPPUNIT_ASSERT_EQUAL(0lu, downgrade_set.size());
+    CPPUNIT_ASSERT_EQUAL(0lu, remove_set.size());
+    CPPUNIT_ASSERT_EQUAL(0lu, obsoleted_set.size());
+}
+
+void BaseGoalTest::test_remove() {
+    std::filesystem::path repo_path = PROJECT_SOURCE_DIR "/test/libdnf/rpm/repos-data/dnf-ci-fedora/x86_64/wget-1.19.5-5.fc29.x86_64.rpm";
```

Comment:
It is not repository path. Better names `package_path`, `pkg_path`, `rpm_path`, ...